sectionproperties/analysis/cross_section.py

```diff
@@ -834,16 +819,14 @@ def calc_plastic():
             plastic_section = PlasticSection(self.geometry, debug)
             plastic_section.calculate_plastic_properties(self, verbose)
 
-        if time_info:
+        if self.time_info:
             text = "--Calculating plastic properties...\n"
             solver.function_timer(text, calc_plastic)
             print("")
```

Comment:
Consider removing these print statements and instead add `\n` to the end of the solver.function_timer function print statement where the time is printed. 

This would add a blank line between any time outputs and get rid of all the `print("")` lines? Currently it's a bit inconsistent, where some `self.time_info` statements have a blank line added using `print("")` and others do not. I think this would tidy up output nicely.

Reply:
Yes, I agree. I will do this instead.